webapp/channels/src/components/admin_console/admin_definition.jsx

```diff
@@ -36,7 +36,7 @@ import ExternalLink from 'components/external_link';
 
 import OpenIdConvert from './openid_convert';
 import Audits from './audits';
-import CustomURLSchemesSetting from './custom_url_schemes_setting.jsx';
+import CustomURLSchemesSetting from './custom_url_schemes_setting.tsx';
```

Comment:
I believe you dont need to explicitly mention `tsx`

Reply:
Thank you for the review. Updated the import.